packages/fluentui/docs/src/examples/components/Dropdown/BestPractices/DropdownBestPractices.tsx

```diff
@@ -1,11 +1,17 @@
 import * as React from 'react';
 
 import ComponentBestPractices from '../../../../components/ComponentBestPractices';
+import { link } from '../../../../utils/helpers';
+import { Text } from '@fluentui/react-northstar';
 
 const doList = [
   'Provide a function that returns the string equivalent of your item if it is an object using the `itemToString` prop',
   'Provide `getA11ySelectionMessage`, `getA11yStatusMessage`, `noResultsMessage` and `loadingMessage` props to visualize dropdown state correctly.',
   'Provide `aria-label` to `triggerButton` slot for non-searchable variants if the placeholder prop is not used.',
+  <Text>
+    Provide corresponding "getA11yStatusMessage" if `loadingMessage` prop is used. Refer to{' '}
```

Comment:
```suggestion
    Provide corresponding "getA11yStatusMessage" if `loadingMessage` prop is used. Refer to 
```

Reply:
@assuncaocharles 
if I remove the empty space then there is no space between the "to" and link. Please see screenshot below:
![image](https://user-images.githubusercontent.com/22620150/84382033-4e258480-abea-11ea-831f-087c872e266c.png)

Reply:
It's only nit but you could do like this:

```Provide corresponding "getA11yStatusMessage" if `loadingMessage` prop is used. Refer to
    {link(' loading example', '#state-loading')} for details.```

Reply:
I see  add the space to the link content itself. It would work as well.
But rather I would leave it how it is, it is more visible what is going on.